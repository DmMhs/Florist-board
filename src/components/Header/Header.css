.Header .info span {
    margin: 7px;
}