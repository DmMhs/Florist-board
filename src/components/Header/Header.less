.Header {
  .info {
    font-style: italic;
    span {
      margin: 7px;
    }
  }
}
