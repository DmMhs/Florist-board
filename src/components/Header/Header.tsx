import React from 'react';

import Navigation from './Navigation/Navigation';
import './Header.less';

const header = () => {
  return (
    <div className="Header">
      <p className="info">
        <span>FLORIST.UA</span>
        <span>MARKET SQUARE, LVIV, UA</span>
        <span>23523</span>
        <span>+123-45-678</span>
      </p>
      <Navigation />
    </div>
  );
};

export default header;
