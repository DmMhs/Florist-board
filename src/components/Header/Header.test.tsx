import React from 'react';
import { shallow, mount } from 'enzyme';

import Header from './Header';

it('Header component matches a snapshot', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
