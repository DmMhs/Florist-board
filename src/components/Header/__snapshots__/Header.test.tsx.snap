// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches a snapshot 1`] = `
<div
  className="Header"
>
  <withRouter(Navigation) />
</div>
`;
